parsers/regex/regex_test.go

```diff
@@ -136,25 +159,52 @@ var tlms = []testLineMap{
 	},
 	{
 		// Sample nginx error log line
-		lineRegex: `(?P<time>\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}) \[(?P<status>.*)\].* request: "(?P<request>[^"]*)"`,
-		input:     `2017/11/07 22:59:46 [error] 5812#0: *777536449 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /isbns HTTP/1.1", upstream: "http://127.0.0.1:8080/isbns", host: "localhost"`,
+		lineRegexes: []string{
+			`(?P<time>\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}) \[(?P<status>.*)\].* request: "(?P<request>[^"]*)"`,
+		},
+		input: `2017/11/07 22:59:46 [error] 5812#0: *777536449 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /isbns HTTP/1.1", upstream: "http://127.0.0.1:8080/isbns", host: "localhost"`,
 		expected: map[string]interface{}{
 			"time":    "2017/11/07 22:59:46",
 			"status":  "error",
 			"request": "GET /isbns HTTP/1.1",
 		},
 	},
+	{
+		// Multi-regex parsing
+		lineRegexes: []string{
+			`(?P<word1>\w+) (?P<word2>\w+) (?P<word3>\w+)`,
+			`(?P<dummy>banana)`,
+		},
+		input: `apple banana orange`, // Should match to first regex
+		expected: map[string]interface{}{
+			"word1": "apple",
+			"word2": "banana",
+			"word3": "orange",
+		},
+	},
+	{
+		// Multi-regex parsing
+		lineRegexes: []string{
+			`\[(?P<word1>\w+)\]`,
+			`(?P<dummy>banana)`,
+		},
+		input: `apple banana orange`, // Should match to second regex
+		expected: map[string]interface{}{
+			"dummy": "banana",
+		},
+	},
```

Comment:
could we add a test for a line that matches none of the listed regexes and see it fall through?